GramAddict/plugins/interact_usernames.py

```diff
@@ -105,6 +105,7 @@ def job():
                     stories_percentage,
                     int(self.args.follow_percentage),
                     int(self.args.follow_limit) if self.args.follow_limit else None,
+                    self.args.scraping_mode,
```

Comment:
why would we scrape a list of usernames to get a list of usernames? 🤔

Reply:
maybe you got a list from someone else and you want to filter it..
or you generated a list and you want to re-scrape with other filters..
that's what I thought :)

Reply:
🤷🏻‍♂️ Okay